packages/shared/src/components/modals/ResponsiveModal.module.css

```
@@ -17,7 +17,8 @@
   & :global(.modal) {
     position: absolute;
     max-width: 30rem;
-    min-height: 100%;
+      max-height: 40rem;
+      min-height: 100%;
```

Just gotta remove the extra indents, I think.
```suggestion
    max-height: 40rem;
    min-height: 100%;
```